{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -1,26 +1,29 @@
-Django==2.1.7
-psycopg2-binary==2.7.7
+-e git+https://github.com/SFDO-Tooling/sfdo-template-helpers.git@v0.7.0#egg=sfdo-template-helpers{% if cookiecutter.use_bucketeer_aws_for_file_storage == 'y' %}
+boto3==1.9.156{% endif %}
+channels_redis==2.4.0
+channels==2.2.0
 dj-database-url==0.5.0
-whitenoise==4.1.2
-channels==2.1.7
-channels_redis==2.3.3
-service_identity==18.1.0
-django-allauth==0.38.0
-werkzeug==0.14.1
-pyopenssl==19.0.0
-django-js-reverse==0.8.2
-djangorestframework==3.9.1
-django-redis==4.10.0
+django-allauth==0.39.1
+django-hashid-field==2.1.6
+django-js-reverse==0.9.1
 django-log-request-id==1.3.2
+django-model-utils==3.1.2
+django-parler==1.9.2
+django-redis==4.10.0
+django-rq-scheduler==1.1.3
+django-rq==1.2.0  # pyup: <1.3.0{% if cookiecutter.use_bucketeer_aws_for_file_storage == 'y' %}
+django-storages==1.7.1{% endif %}
+Django==2.2.1
+djangorestframework==3.9.4
+logfmt==0.4
+newrelic==4.20.0.120
+Pillow==6.0.0
+psycopg2-binary==2.8.2
+pyopenssl==19.0.0
 raven==6.10.0
-rq==0.12  # pyup: <0.13
-django-rq==1.2.0  # pyup: <1.3.0
-django-rq-scheduler==1.1.1
-rq-scheduler==0.8.3  # pyup: ==0.8.3{% if cookiecutter.use_bucketeer_aws_for_file_storage == 'y' %}
-django-storages==1.7.1
-boto3==1.9.96{% endif %}
-django-hashid-field==2.1.5
 redis==2.10.6  # pyup: <3.0
-logfmt==0.4
--e git+https://github.com/SFDO-Tooling/sfdo-template-helpers.git@v0.3.0#egg=sfdo_template_helpers
-django-parler==1.9.2
+rq-scheduler==0.8.3  # pyup: ==0.8.3
+rq==0.12  # pyup: <0.13
```

Comment:
@wlonk do you remember what the blocker was for updating redis, rq, rq-scheduler, and django-rq? We're using the latest versions for MetaCI.

Reply:
No, I don't. I just remember being surprised by it.